### ISSUE TITLE
Missing Users Table in Database

### ISSUE THREAD

**Issue by snoski3** [Snipa22/nodejs-pool#207]
I spoke too soon.  I'm missing is the users table.  
Can I manually create the table?

![image](https://user-images.githubusercontent.com/5173575/34045447-84ffa506-e16e-11e7-8309-c3bb164989a4.png)

Thanks,

**Comment by rowanruseler** [comment]
You certainly can:

```
CREATE TABLE `users` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `username` varchar(256) NOT NULL,
  `pass` varchar(64) DEFAULT NULL,
  `email` varchar(256) DEFAULT NULL,
  `admin` tinyint(1) DEFAULT '0',
  `payout_threshold` bigint(16) DEFAULT '0',
  `enable_email` tinyint(1) DEFAULT '1',
  PRIMARY KEY (`id`),
  UNIQUE KEY `users_id_uindex` (`id`),
  UNIQUE KEY `users_username_uindex` (`username`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

**Comment by mtompkins** [comment]
FWIW there is also a xmrtoTxn table in mine

**Comment by snoski3** [comment]
@rowanruseler Thank you for your help.  I finally was able to run the command, however I received an error:

![image](https://user-images.githubusercontent.com/5173575/34074422-5471e3b6-e274-11e7-9c25-6440328ce381.png)

Any thoughts?

**Comment by rowanruseler** [comment]
@snoski3 yeah, you've a miss configured MySQL. What version are you running?

**Comment by snoski3** [comment]
@rowanruseler
Version 5.6.37

**Comment by rowanruseler** [comment]
@snoski3 I think your issue is related to character and collation of the server configuration. But I can't be certain unless you provide the additional information requests above.

**Comment by snoski3** [comment]
@rowanruseler
According to the link below, it is a limitation of MySQL 5.6, I should upgrade to 5.7. 
I will upgrade and keep everyone posted. 

https://stackoverflow.com/questions/1814532/1071-specified-key-was-too-long-max-key-length-is-767-bytes

**Comment by rowanruseler** [comment]
@snoski3 it is most likely related to your character and collation settings within my.cnf. Post the output of: `show global variables where Variable_name like 'character%';` and I can be certain.

**Comment by snoski3** [comment]
@rowanruseler 
Here is the output:

![screenshot_20171219-053659](https://user-images.githubusercontent.com/5173575/34155468-08c309d8-e47f-11e7-9f4a-97b64c3e9bdb.png)

Thanks,

**Comment by rowanruseler** [comment]
@snoski3 use varchar(255) for the email part, and it should work. Or indeed upgrade towards the newer verison.

**Comment by snoski3** [comment]
@rowanruseler 
I made a small adjustment and it worked. I used 255 for the username and email.

![screenshot_20171219-064254](https://user-images.githubusercontent.com/5173575/34157635-0f208108-e488-11e7-9eac-a22b5c3d31cc.png)

Thank you very much for your help!!!
Now I need to create an account.

**Comment by snoski3** [comment]
I deleted all of the folders so I can start from scratch.  Once it was finished I connected to the users table and ran select * from users; which resulted in no entries found.  I even created a new user called administrator with the password in the email field and set admin field to 1.
I still can't login to the pool UI.  

Is there a way that I can confirm that the UI is connected to the database?

thanks,
